$OpenBSD: patch-libaccounts-glib_ag-manager_h,v 1.1.1.1 2013/03/29 13:16:17 jasper Exp $

Don't redefine AgManager which is already declared in ag-types.h.

--- libaccounts-glib/ag-manager.h.orig	Fri Mar 29 13:43:09 2013
+++ libaccounts-glib/ag-manager.h	Fri Mar 29 13:53:17 2013
@@ -45,7 +45,6 @@ G_BEGIN_DECLS
 
 typedef struct _AgManagerClass AgManagerClass;
 typedef struct _AgManagerPrivate AgManagerPrivate;
-typedef struct _AgManager AgManager;
 
 /**
  * AgManagerClass:
